packages/web/src/app/doc-pages/about/tutorial/angular-tutorial/angular-tutorial.component.html

```diff
@@ -0,0 +1,115 @@
+<!--BASIC TUTORIAL-->
+<h3 class="e-title-small e-mt-40 e-mb-0">Set up the basics</h3>
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <div class="e-title-xsmall">Install CSS library</div>
+    <p class="e-mt-8">
+      Install the
+      <a [routerLink]="['/components/css-library']" class="e-link e-link--inline">CSS library</a> to get the
```

Comment:
This is a very tiny comment which you can ignore if you want to, but when we pass static primitive values as props, we don't need to use [] around the prop, e.g.
``` html
[routerLink]="['/components/css-library']"
```
can be written as
``` html
routerLink="/components/css-library"
```
I find the latter much more readable. This also goes for line 16 for example, where we can write `fileName="Terminal"` instead of `[fileName]="'Terminal'"` and so forth 🙂

Reply:
Be aware that this only goes for primitive types. When we pass arrays (such as `tabs` on line 16), we need to pass it as an array, and therefore the prop needs to be wrapped in [] to make it a prop instead of an attribute.